bin/grunt

```diff
@@ -4,41 +4,53 @@
 
 process.title = 'grunt';
 
-// Especially badass external libs.
-var findup = require('findup-sync');
-var resolve = require('resolve').sync;
-
-// Internal libs.
-var options = require('../lib/cli').options;
-var completion = require('../lib/completion');
-var info = require('../lib/info');
-var path = require('path');
-
-var basedir = process.cwd();
-var gruntpath;
-
-// Do stuff based on CLI options.
-if ('completion' in options) {
-  completion.print(options.completion);
```

Comment:
Are we going to drop the `completion` command line flag? If someone was configuring the auto-completion as [pointed](https://github.com/gruntjs/grunt-cli#shell-tab-auto-completion) in the `README`, they will only see `Grunt` trying to run their tasks directly after opening a new terminal session, which will result in an exception simular to the one in [my other comment](https://github.com/gruntjs/grunt-cli/pull/117/files#r172367757).